### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by cancelei**
Hello Reviewer,

Here you find the repo with screen shorts of the filtered table outputs.

Find all animals whose name ends in "mon".
image

List the name of all animals born between 2016 and 2019.
image

List the name of all animals that are neutered and have less than 3 escape attempts.
image

List the date of birth of all animals named either "Agumon" or "Pikachu".
image

List name and escape attempts of animals that weigh more than 10.5kg
image

Find all animals that are neutered.
image

Find all animals not named Gabumon.
image

Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equal precisely 10.4kg or 17.3kg)
image

Happy review.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,8 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    name varchar(100),
```
Comment (Ghiftee) [inline schema.sql:5]: - [x] According to the requirements below, the animal table should have the following columns but you are missing the `id` column. It is important to have an `id` for easier distinction of animals. 
```
Create a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,23 @@
-/*Queries that provide answers to the questions from all projects.*/
+-- Find all animalswhose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+-- List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (Ghiftee) [inline queries.sql]: - [ ] Great job adding these queries, however, you are required to list just the `name` of these animals and not the entire details so your queries should start with `SELECT name ...`

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @cancelei 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Queries are successful
✔️ Screenshots are added
✔️ SQL commands are put in correct files

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review 👇🏽 _

- [x] Kindly remove the live demo link from your README since this project is not deployed. It is more professional to have a message such as `coming soon` in place of the broken link.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- In addition to including an `id` you can mark it as the primary key. Marking your id as primary key, serves many benefits as you continue to build your database

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md
Comment (KanzaTahreem) [inline README.md:1]: - [x] Well done for implementing most of the suggestions 👏 However, I noticed that you missed one from the review summary. Kindly make sure to fix this 👇

image

 _The link is redirecting to the following_

image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @cancelei 

Great work on making the changes requested by a previous reviewer 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Table is created
- ✅ Successful queries
- ✅ Screenshots were added

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

- [OPTIONAL] I think it would be better to remove the placeholders (everything that starts with `>`) from the README for making it more professional and clean💪

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @cancelei 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**